client/rest/src/test/java/org/elasticsearch/client/HeapBufferedAsyncResponseConsumerTests.java

```diff
@@ -106,7 +106,8 @@ public void testCanConfigureHeapBufferLimitFromOutsidePackage() throws ClassNotF
             IllegalAccessException, InvocationTargetException, InstantiationException {
         int bufferLimit = randomIntBetween(1, Integer.MAX_VALUE);
         //we use reflection to make sure that the class can be instantiated from the outside, and the constructor is public
-        Constructor<?> constructor = HttpAsyncResponseConsumerFactory.HeapBufferedResponseConsumerFactory.class.getConstructor(Integer.TYPE);
+        Constructor<?> constructor = HttpAsyncResponseConsumerFactory.HeapBufferedResponseConsumerFactory.class
+                .getConstructor(Integer.TYPE);
```

Comment:
Does it fit if you wrap after the equals?

Reply:
Sure! I'll switch it.